assets/i18n/qqq.json

```diff
@@ -285,6 +285,7 @@
 	"I18N_LIBRARY_VIEW_ALL": "Text of a button shown to the side of the exploration group. - When clicked, it shows all the explorations included on the associated group. It should be a small text.\n{{Identical|View all}}",
 	"I18N_ONE_SUBSCRIBER_TEXT": "Text displayed under the subscribers tab in creator dashboard. If the creator has one subscriber, this text is displayed which informs him/her about the same.",
 	"I18N_PLAYER_BACK": "Text read to users with screenreaders when they navigate through an exploration. - This labels the leftward-pointing arrow that is used to go backward by one card in the exploration.\n{{Identical|Back}}",
+	"I18N_PLAYER_BACK_TO_COLLECTION": "Text shown to users as they navigate through each exploration in a collection - This labels the link that is used to return back to the home page of the collection the user is currently exploring",
```

Comment:
Just to check, isn't this only shown at the end of the exploration (not "as they navigate through it")?

Second sentence looks great!